malaria24/ona/models.py

```diff
@@ -157,14 +165,25 @@ def get_digest_email_data(self):
 
                 if province_cases:
                     start_date = province_cases \
-                        .first().create_date_time.strftime(
-                            "%d %B %Y"
-                        )
+                        .first().create_date_time
+                    if min_date == temp_min_date:
+                        min_date = start_date
+                    else:
+                        if start_date < min_date:
+                            min_date = start_date
```

Comment:
I think there might be a better way to achieve this.

Either, we set `min_date` to `datetime.max` and `max_date` to `datetime.min`. That way, when we do `start_date < min_date`, it will always be true for the first date, and we can get rid of the first `min_date == temp_min_date`, as well as getting rid of `temp_min_date`.

The other option is setting `min_date = None`, and then we can do
```
if min_date is None:
    min_date = start_date
elif start_date < min_date:
    min_date = start_date
```
and then we don't need the `datetime.min`. But then we also have to handle the case where there aren't any cases, so we'll need to check if `start_date` is `None`, and if it is, we'll need to set it to some default date value.